package.json

```diff
@@ -224,7 +224,7 @@
         },
         {
           "command": "vscode-ibmi-projectexplorer.removeFromIncludePaths",
-          "when": "view == projectExplorer && viewItem =~ /^includePath.*/",
+          "when": "view == projectExplorer && viewItem =~ /^includePath(?!s).*/",
```

Comment:
what does this do?

Reply:
`?!` is the regex notation for negative look ahead. The purpose of this is for this action to appear on tree items with a `contextValue` of `includePath` and not `includePaths` which is only used for the Include Paths heading.